lib/WP_Auth0_LoginManager.php

```diff
@@ -232,8 +228,7 @@ public function implicit_login() {
 		$id_token_param = ! empty( $_POST['id_token'] ) ? $_POST['id_token'] : $_POST['token'];
 		$id_token       = sanitize_text_field( wp_unslash( $id_token_param ) );
 
-		$jwt_verifier  = new WP_Auth0_Id_Token_Validator( $id_token, $this->a0_options );
-		$decoded_token = $jwt_verifier->decode( true );
+		$decoded_token = $this->decode_id_token( $id_token );
 		$decoded_token = $this->clean_id_token( $decoded_token );
```

Comment:
better to return token as is and not let the library modify its contents

Reply:
I'm not changing how the token is used in this PR, only how it's validated.